react/features/base/settings/reducer.js

```diff
@@ -0,0 +1,157 @@
+// @flow
+import _ from 'lodash';
+
+import { APP_WILL_MOUNT } from '../../app';
+
+import JitsiMeetJS, { browser } from '../lib-jitsi-meet';
+import { ReducerRegistry } from '../redux';
+import { PersistenceRegistry } from '../storage';
+import { assignIfDefined, randomHexString } from '../util';
+
+import { SETTINGS_UPDATED } from './actionTypes';
+
+const logger = require('jitsi-meet-logger').getLogger(__filename);
+
+/**
+ * The default/initial redux state of the feature {@code base/settings}.
+ *
+ * @type Object
+ */
+const DEFAULT_STATE = {
+    audioOutputDeviceId: undefined,
+    avatarID: undefined,
+    avatarURL: undefined,
+    cameraDeviceId: undefined,
+    displayName: undefined,
+    email: undefined,
+    localFlipX: true,
+    micDeviceId: undefined,
+    serverURL: 'https://meet.jit.si',
```

Comment:
Not really, see original implementation: https://github.com/jitsi/jitsi-meet/blob/740817fdcfca7381d057f427cecf9f411839f84e/react/features/app/components/AbstractApp.js#L388

I basically just relocated the constant, as it makes more sense to me that it's in the settings rather than in a React component. What do you think?

Reply:
Ack.